packages/front-end/components/Experiment/AlignedGraph.tsx

```diff
@@ -56,6 +57,8 @@ const percentFormatter = new Intl.NumberFormat(undefined, {
   maximumFractionDigits: 0,
 });
 
+const numberFormatter = Intl.NumberFormat();
```

Comment:
Do we need this sort of thing here?
```
  const formatterOptions: Intl.NumberFormatOptions = {
    ...(differenceType === "relative" ? { maximumFractionDigits: 1 } : {}),
    ...(differenceType === "scaled" ? { notation: "compact" } : {}),
```

Reply:
I don't want to mess with your tinkering with the formatter, but I'll add a setting for `!percent` that checks if the width is > 5000 and then sets the notation to compact.